_includes/snippets_library/ComputeCanada_Graham_slurm/_config_options.yml

```diff
@@ -53,3 +53,13 @@ sched:
   info: "sinfo"
   comment: "#SBATCH"
   hist: "sacct -u yourUsername"
+
+episode_order:
+  - 11-hpc-intro
+  - 12-cluster
+  - 13-scheduler
+  - 14-modules
+  - 15-transferring-files
+  - 16-parallel
+  - 17-resources
+  - 18-responsibility
```

Comment:
Please add a newline. POSIX will thank you.

Reply:
Done.